Exercise_3.py

```diff
@@ -1,32 +1,79 @@
-class ListNode:
-    """
-    A node in a singly-linked list.
-    """
-    def __init__(self, data=None, next=None):
-    
-class SinglyLinkedList:
+#Exercise_3 : Implement Singly Linked List
+class Node:
+    def __init__(self,initdata):
+        self.data=initdata
+        self.next=None
+
+    def set_data(self,newdata):
+        self.data=newdata
+
+    def set_next(self,newnode):
+        self.next=newnode
+
+    def get_data(self):
+        return self.data
+
+    def get_next(self):
+        return self.next
+
```

Comment:
Good! Please try to write code without these methods as the methods increase the overhead of function calls.